src/Cohesity.Powershell/Scripts/ProtectionPolicy/New-CohesityProtectionPolicy.ps1

```diff
@@ -45,6 +50,28 @@ function New-CohesityProtectionPolicy {
 
         $headers = @{'Authorization' = 'Bearer ' + $token }
 
+        $snapshotArchivalCopyPolicies = $null
+        if ($VaultName) {
+            $resp = Get-CohesityVault -VaultName $VaultName
+            if ($null -eq $resp) {
+                Write-Host "Vault (external target) name '$VaultName' does not exists"
+                return
+            }
+            # Except the vault info all the default values referenced from cohesity UI
+            $snapshotArchivalCopyPolicies = @(
+                @{
+                    copyPartial = $true
+                    daysToKeep  = 90
```

Comment:
Wont 'daysToKeep' value change based on 'RetainInDays' parameter value?

Reply:
For archival (on the vault) the retain value could be different from the snapshot protection in the cluster.